polaris-react/src/components/IndexFilters/components/SortButton/SortButton.tsx

```diff
@@ -123,6 +123,7 @@ export function SortButton({
       activator={sortButton}
       autofocusTarget="first-node"
       onClose={handleClose}
+      preferredAlignment={se23 ? 'right' : undefined}
```

Comment:
(We can leave out the conditional here since this is polish we want regardless)

```suggestion
      preferredAlignment="right"
```